### PR TITLE
Apply styling to trials#index

### PR DESCRIPTION
**Pull request by mwenger1**
Rather than applying new styles and fighting the existing template, it
was easier to model this page off of a few existing pages on the
National Brain Tumor site and work with the existing class structure.

Pages for Reference:
http://braintumor.org/join-the-fight/ways-to-give/
http://braintumor.org/join-the-fight/volunteer/

https://github.com/mwenger1/trial-match/issues/17

STILL TO DO:

* Style form inputs
* Trim description text with ellipsis

screen shot 2016-08-14 at 8 29 49 pm

### CODE REVIEW

**Comment by mwenger1** [comment]
@opihana First pass at beginning the styling

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,9 +1,22 @@
 @charset "utf-8";
 
-@import "normalize-rails";
+.sidebar {
+  float: left;
+}
 
-@import "bourbon";
-@import "neat";
```
Comment (opihana) [inline app/assets/stylesheets/application.scss]: Why are we getting rid of these?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+.sidebar:before {
+  left: 0;
+  z-index: -1;
+}
 
-@import "base/base";
-@import "refills/flashes";
+main {
+  padding-right: 0;
```
Comment (opihana) [inline app/assets/stylesheets/application.scss]: What styles are we overriding? Where are they coming from?

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+}
+
+.trial-count {
+  text-align: right;
+}
+
+.trial-list li {
+  border-bottom: 1px solid #dddddd;
```
Comment (opihana) [inline app/assets/stylesheets/application.scss]: Do you have a variables file to start on the color pallette?

**Comment by opihana** [comment]
As we discussed over the phone, 🚢 without my suggestions and when can refactor when I'm back.

**Comment by opihana** [comment]
@mwenger1 check out the latest commit! Let me know your thoughts.

Changes:

- added back Bourbon/Bitters within the trials' id, similar to how we do in the check-in app. (Even though this isn't a great practice for regular apps, since this is a 'plug-in' style, best not to create conflicting/overlapping styles, but the features of those gems are still desirable in our section)
- using variables
- addd base styles for form inputs
- `simple_form` configs for form inputs
- fixed an accident from the previous commit on this PR that was breaking the test suite

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -1,9 +1,43 @@
...
+  .form-actions {
+    @include margin($base-spacing null);
+  }
+
+  .trial-results {
+    @include padding($large-spacing);
+    flex: 2;
+  }
+
+  .sidebar::before {
```
Comment (mwenger1) [inline app/assets/stylesheets/application.scss]: What do you think about changing this to use `.trial-sidebar` and we remove `.sidebar` from the markup?

File: app/views/trials/_search_filter.html.erb [hunk 5]
```diff
@@ -1,27 +1,39 @@
...
+          collection: ["Male", "Female"]
+        )
+      ) %>
+
+      <%= f.input(
+        :control,
+        filter_radio_default(:control).merge(
+          as: :radio_buttons,
+          collection: control_options,
+          label: "Patient or control?",
```
Comment (mwenger1) [inline app/views/trials/_search_filter.html.erb]: What do you think about moving this to i18n

**Comment by mwenger1** [comment]
🚢 Nicely done!

**Pull request merged**